cloud_sql/mysql_instance_read_pool/main.tf

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# [START cloud_sql_mysql_read_pool]
+
+resource "google_sql_database_instance" "primary" {
+  name             = "mysql-primary"
+  database_version = "MYSQL_8_0"
+  region           = "europe-west4"
+
+  instance_type = "CLOUD_SQL_INSTANCE"
+
+  # If you wish to, set deletion_protection to true to prevent
+  # accidental deletion of this instance as a result of removing the
+  # Terraform resource.
+  deletion_protection = false
```

Comment:
We've added functionality to this repo so that _in the context of CI_, this setting is not required. 

This sample is not being applied anyway, so this value is not needed for CI.
```suggestion
```